### PR TITLE
Notebook 4.2

### PR DESCRIPTION
**Pull request by bollwyvl**
> Don't merge until notebook 4.2 is released!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,32 +27,22 @@ When you are happy with your presentation, you can download the standalone HTML
...
 ## Install
 
-> Note: installing directly off this repo won't work, as we don't ship the built JavaScript and CSS assets. See more about [developing](#develop) below.
+> Note: installing directly off the `git` repo won't work, as we don't ship the built JavaScript and CSS assets. See more about [developing](#develop) below.
 
 ### `pip`
 ```shell
 pip install nbpresent
-python -m nbpresent.install
+jupyter nbextension install --py=nbpresent --overwrite --user
```
Comment (malev) [inline README.md]: why aren't you sending `--py` to the end of the line?

File: README.rst [hunk 2]
```diff
@@ -58,8 +58,8 @@ standalone HTML file from the *File -> Download as -> Presentation
 Install
```
Comment (malev) [inline README.rst]: why do you have a readme.md and a readme.rst ?

File: conda.recipe/post-link.bat [hunk 3]
```diff
@@ -1 +1,7 @@
-"%PREFIX%\python.exe" -m nbpresent.install --overwrite --enable --prefix="%PREFIX%" && if errorlevel 1 exit 1
+"%PREFIX%\Scripts\pip" install -e git+https://github.com/ellisonbg/notebook.git#egg=notebook
+
+"%PREFIX%\Scripts\jupyter" nbextension install --py="%PKG_NAME%" --overwrite --sys-prefix
```
Comment (malev) [inline conda.recipe/post-link.bat]: `--py` should be at the end of the line

File: conda.recipe/pre-unlink.bat [hunk 4]
```diff
@@ -0,0 +1,4 @@
+"%PREFIX%\Scripts\jupyter" nbextension uninstall --py="%PKG_NAME%" --sys-prefix
+"%PREFIX%\Scripts\jupyter" nbextension disable --py="%PKG_NAME%" --sys-prefix
```
Comment (malev) [inline conda.recipe/pre-unlink.bat]: same as above

File: conda.recipe/post-link.sh [hunk 5]
```diff
@@ -1 +1,3 @@
-"${PREFIX}/bin/python" -m nbpresent.install --overwrite --enable --prefix="${PREFIX}"
+"${PREFIX}/bin/jupyter" nbextension install --py="${PKG_NAME}" --overwrite --sys-prefix
```
Comment (malev) [inline conda.recipe/post-link.sh]: same as above

File: environment42.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+name: nbpresent42
```
Comment (malev) [inline environment42.yml]: why do you call it environment42?

File: environment42.yml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  - cpcloud
+  - javascript
+  - mutirri
+dependencies:
+  - anaconda-client
+  - coverage
+  - flake8
+  - ipywidgets
+  - mock
+  - nb_config_manager
```
Comment (malev) [inline environment42.yml]: do we still need this?

**Comment by bollwyvl** [comment]
Closed elsewise....

**Pull request closed without merging**